File: app/src/main/java/com/example/changfeng/taptapword/net/BaiduFanyiService.java
package com.example.changfeng.taptapword.net;

import com.example.changfeng.taptapword.net.result.BaiduResult;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

/**
 * Created by changfeng on 2016/3/17.
 */
public interface BaiduFanyiService {
    @GET("/api/trans/vip/translate")
    Call<BaiduResult> getResult(@Query("q") String query, @Query("from") String from, @Query("to") String to, @Query("appid") String appId, @Query("salt") String salt, @Query("sign") String sign);
}

File: app/src/main/java/com/example/changfeng/taptapword/net/BaiduFanyiService.kt
package com.example.changfeng.taptapword.net

import com.example.changfeng.taptapword.net.result.BaiduResult

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

/**
 * Created by changfeng on 2016/3/17.
 */
interface BaiduFanyiService {
    @GET("/api/trans/vip/translate")
    fun getResult(@Query("q") query: String, @Query("from") from: String, @Query("to") to: String, @Query("appid") appId: String, @Query("salt") salt: String, @Query("sign") sign: String): Call<BaiduResult>
}
